public/javascripts/dashboard.js

```diff
@@ -119,41 +119,69 @@ app.controller('fc', function ($scope, $http) {
             $scope.$evalAsync();
             console.log($scope.candidate.first_name)
         })
-});
 
-app.controller('mc', function ($scope, $http) {
     $scope.s = window.s;
+    $scope.applying = false;
+    $scope.cur_candidate = {};
+
+
+    $scope.setCurCandidates = function (value) {
+        console.log('updating');
+        $scope.cur_candidate = value;
+        console.log($scope.cur_candidate)
+        console.log($scope.cur_candidate.first_name);
+
+        $scope.$evalAsync();
+    }
     $http.get('/dashboard/vacancies')
         .then(function (data) {
             console.log(data)
             $scope.vacancies = data.data;
             console.log($scope.vacancies)
         })
     $scope.selectvc = function (val) {
-        console.log(val);
+        $(document).ready(function(){
+            $('#apply').removeClass('hide');
+            $('ul.tabs').tabs();
+            $('#apply').click();
+        })
         $scope.cur_vacancy = val;
-        $(document).ready(function () {
-            $http.post('/dashboard/app_data', JSON.stringify({vid : val}))
+        $scope.$evalAsync();
+
+        if($scope.cur_vacancy) {
+            console.log('inside')
+            $http.post('/dashboard/app_data', JSON.stringify({vid : $scope.cur_vacancy}))
                 .then(function (data) {
-                    console.log(data)
-                    if(data.data.length == 1){
+                    if(data.data.length == 1) {
                         $scope.cur_candidate = mapper(data.data[0]);
+                        console.log('length 1')
+                        console.log($scope.cur_candidate)
+                        $scope.setCurCandidates(mapper(data.data[0]));
+
                     } else {
                         $scope.cur_candidate = $scope.candidate;
                     }
+                    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/javascripts/dashboard.js
+++ b/public/javascripts/dashboard.js
@@ -161,7 +161,7 @@
                     } else {
                         $scope.cur_candidate = $scope.candidate;
                     }
-                    
+
 
                 })
         }
```